### ISSUE TITLE
ROCK 4C not mentioned in README.md - what system image to use?

### ISSUE THREAD

**Issue by 3col** [radxa/debos-radxa#37]
Please could you confirm, as ROCK 4C is not mentioned in README.md, does it mean we can use ROCK 4C+ images instead?
E.g.  rock-**4c-plus**-debian-bullseye-xfce4-arm64-20221031-1305-gpt.img.xz  works for "older" ROCK 4C too - right or not?

**Comment by jack-ma** [comment]
No. 4C+ and 4C are different hardware. Please check the rbuild repo, debos is not planned to maintain in the future. 4C is supported in the rbuild.

**Comment by 3col** [comment]
Do you mean 4C should use this repo https://github.com/radxa-repo/rbuild  ?

**Comment by jack-ma** [comment]
Yes, and the image will be generated at:

https://github.com/radxa-build

No 4C images release yet, but it will land there soon.
